### ISSUE TITLE
 "load in new tab" -> "user defined" doesn't work 

### ISSUE THREAD

**Issue by helppe** [yfdyh000/tabutils#69]
hi, 
i use the last version of firefox (43.0.4) and when i enter an url adress in the "load in new tab" -> "user defined" choice, (for example google), when i open a new tab there is nothing at all, no url is loaded at all.
can you confirm the issue please? thanks

**Comment by yfdyh000** [comment]
Yes, it will not work since https://bugzilla.mozilla.org/show_bug.cgi?id=1118285 (Firefox 41).
Use the https://addons.mozilla.org/firefox/addon/new-tab-override/.

**Comment by helppe** [comment]
it isn't possible to fix it rather than install an other addon?
the less addons we install, the better it is for our browser, isnt it?^^

**Comment by yfdyh000** [comment]
Yes, it is possible, but I have no energy to study it. I welcome any patch for it.

**Comment by helppe** [comment]
ok thanks.
